torch/nn/functional.py

```diff
@@ -662,11 +662,7 @@ def dropout3d(input, p=0.5, training=True, inplace=False):
         training: apply dropout if is ``True``. Defualt: ``True``
         inplace: If set to ``True``, will do this operation in-place. Default: ``False``
     """
-    if p < 0 or p > 1:
-        raise ValueError("dropout probability has to be between 0 and 1, "
-                         "but got {}".format(p))
-    f = _VF.feature_dropout_ if inplace else _VF.feature_dropout
-    return f(input, p, training)
+    return dropout2d(input, p, training, inplace)
```

Comment:
I would not change to calling dropout2d here (yes I know it's the same code). It will be confusing if it errors and users see dropout2d in their trace.